backend/src/plugins/Utility/commands/CleanCmd.ts

```diff
@@ -33,9 +33,15 @@ export async function cleanMessages(
   idsToDelete.forEach((id) => pluginData.state.logs.ignoreLog(LogType.MESSAGE_DELETE, id));
   pluginData.state.logs.ignoreLog(LogType.MESSAGE_DELETE_BULK, idsToDelete[0]);
 
-  // Actually delete the messages
-  channel.bulkDelete(idsToDelete);
-  await pluginData.state.savedMessages.markBulkAsDeleted(idsToDelete);
+  // Actually delete the messages (in chunks of 100)
+
+  const chunks = chunkArray(idsToDelete, 100);
+  await Promise.all(
+    chunks.map(async (chunk) => {
+      await channel.bulkDelete(chunk);
+      await pluginData.state.savedMessages.markBulkAsDeleted(chunk);
+    }),
```

Comment:
blazing fast promise master race 🏎️🚀
```suggestion
    chunks.map((chunk) => Promise.all([
      channel.bulkDelete(chunk),
      pluginData.state.savedMessages.markBulkAsDeleted(chunk),
    ]),
```
probably works